pages/apim/3.x/user-guide/publisher/plans/plan-policies.adoc

```diff
@@ -0,0 +1,26 @@
+= Configure plan policies
+:page-sidebar: apim_3_x_sidebar
+:page-permalink: apim/3.x/apim_publisherguide_plan_policies.html
+:page-folder: apim/user-guide/publisher
+:page-keywords: Gravitee.io, API Platform, API Management, API Gateway, documentation, manual, guide, reference, api, CGU, GCU
+:page-layout: apim3x
+
+== Overview
+
+NOTE: From APIM 3.5.x, the recommended method for adding policies to plans is with link:/apim/3.x/apim_publisherguide_design_studio_create.html#flow-policies[Design Studio^]
+
+You can configure a single policy, or a chain of policies. Policy chains can be of the same or different types.
+
+== Configure policies
+
+You configure policies on the **Policies** page of a new or existing plan.
+
+. To edit an existing plan, select your API and click **Portal > Plans**.
+. On the **Policies** page:
+.. Choose a policy type from the list and click *ADD*.
++
+image::apim/3.x/api-publisher-guide/plans-subscriptions/add-plan-policies.png[]
++
+.. Depending on the policy type, specify any other required details of the policy.
+.. If you want to create a policy chain, repeat the same steps for each policy in the chain.
+. Click **SAVE** to save your plan.
```

Comment:
I think this screen only exists for v1 API
I don't know if we need to add this to documentation
wdyt ?